client/verta/verta/_deployment/endpoint.py

```diff
@@ -213,3 +213,9 @@ def get_access_token(self):
         if len(tokens) == 0:
             return None
         return tokens[0]['creator_request']['value']
+
+    def form_update_body(self, resources, strategy, build_id):
+        update_body = strategy._as_build_update_req_body(build_id)
+        update_body["resources"] = reduce(lambda resource_a, resource_b: {**resource_a, **resource_b}, map(lambda resource: resource.to_dict(), resources))
```

Comment:
```python
{**resource_a, **resource_b}
```
This syntax is not supported in Python 2. I think we will need a different way to merge dicts.

Reply:
replaced to python2 syntax